injecteddll/src/lib.rs

```diff
@@ -1,20 +1,94 @@
 // lib.rs
-windows::include_bindings!();
 
-use crate::Windows::Win32::Foundation::HINSTANCE;
-use crate::Windows::Win32::System::SystemServices::*;
+use bindings_dll::Windows::Win32::System::Threading::*;
+use bindings_dll::Windows::Win32::Foundation::*;
+use bindings_dll::Windows::Win32::System::ProcessStatus::{K32EnumProcessModules, K32GetModuleFileNameExA};
+use bindings_dll::Windows::Win32::System::SystemServices::*;
+
 use core::ffi::c_void;
+use std::convert::TryInto;
+use std::mem;
+use std::u32;
+use std::str;
+use std::fmt;
+use winsafe_dll::{winexec, SafeHandle};
+
+#[derive(Debug)]
```

Comment:
Don't copy paste code, if needed we can create a common crate to share code to other crates